package.json

```diff
@@ -59,6 +59,7 @@
     "eslint-plugin-react-app": "^4.0.1",
     "eslint-plugin-react-hooks": "^1.5.1",
     "flux": "^2.0.1",
+    "immutable": "4.0.0-rc.12",
```

Comment:
Worth noting we don't use this verison internally. Plus since it's RC could use latest v3

Reply:
good catch. just did a `-DE`, didn't think about the fact that we're not on latest